src/tag.cpp

```diff
@@ -369,7 +376,6 @@ int HSTag::cycleAllCommand(Input input, Output output)
             frame->cycleAll(FrameTree::CycleDelta::Begin, skip_invisible);
         } else {
             floating_clients_focus_ = static_cast<size_t>(newIndex);
-            floating_clients_[floating_clients_focus_]->raise();
```

Comment:
Just trying to make sure: This will be taken care of in the caller of cycleAllCommand, right?

Reply:
There is a `newFocus->raise()` in line 402 at the end of the `cycleAllCommand()` function. I should have mentioned that in the commit description already. (the commit/pr description mentions this now)

Reply:
Awesome, thanks for adding it to the commit message!